code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -0,0 +1,85 @@
+/turf/simulated/floor/wood
+	icon_state = "wood"
+	floor_tile = /obj/item/stack/tile/wood
+	broken_states = list("wood-broken", "wood-broken2", "wood-broken3", "wood-broken4", "wood-broken5", "wood-broken6", "wood-broken7")
+
+	footstep_sounds = list(
+		"human" = list('sound/effects/footstep/wood_all.ogg'), //@RonaldVanWonderen of Freesound.org
+		"xeno"  = list('sound/effects/footstep/wood_all.ogg')  //@RonaldVanWonderen of Freesound.org
+	)
+
+/turf/simulated/floor/wood/attackby(obj/item/C, mob/user, params)
+	if(..())
+		return
+	if(istype(C, /obj/item/weapon/screwdriver))
+		if(broken || burnt)
+			return
+		user << "<span class='danger'>You unscrew the planks.</span>"
+		new floor_tile(src)
```

Comment:
That's the weird thing, most of this is copypasted from -tg-; There are also oddities in the parent call

```
if(..())
    return 1
```
```
if(!..())
    return 0
```
```
if(..())
    return
```

All of those are from different `floor/` subtype `attackby()`s.